packages/user/style/menu.css

```
@@ -3,8 +3,23 @@
 | Distributed under the terms of the Modified BSD License.
 |----------------------------------------------------------------------------*/
 
+.lm-MenuBar-item[data-empty='true'],
+.p-MenuBar-item[data-empty='true'] {
```

Let's drop the old phosphor class as we know we are using recent version

```suggestion
.lm-MenuBar-item[data-empty='true'] {
```